### PR TITLE
Fixes kube config output writing to console

### LINKED ISSUE #802: kube_config_raw revealed in Build Logs
**Describe the bug:**
Cluster credentials are printed out to the console in the build logs.

**To Reproduce:**
Run `terraform apply` and see the output logs.

**Expected behavior:**
Cluster credentials are saved to the output config without being displayed to the console.

**Screenshots:**
```
module.aks.null_resource.cluster_credentials[0]: Creating...
module.aks.null_resource.cluster_credentials[0]: Provisioning with 'local-exec'...
module.aks.null_resource.cluster_credentials[0] (local-exec): Executing: ["/bin/sh" "-c" "if [ ! -e ./output ]; then mkdir -p ./output; fi && echo \"apiVersion: v1\nclusters:\n- cluster:\n    certificate-authority-data: ...\n    server: https://....hcp.eastus.azmk8s.io:443\n  name: ... \ncontexts:\n- context:\n    cluster: A...\n    user: ...\n  name: ...\nkind: Config\npreferences: {}\nusers:\n- name: ...\n  user:\n    client-certificate-data: ...\n    client-key-data: ...\n    token: ...\n\" > ./output/kube_config"]
module.aks.null_resource.cluster_credentials[0]: Creation complete after 0s [id=6020571218102932175]
```

**Additional context:**

**Does this require updates to documentation?:**
I don't think it would be desired behavior. There should a big red warning in the code if it's the desired behavior.

### PR DESCRIPTION
**Pull request by sarath-p**
Closes #802 

Module source links changed to relative path for testing, but will change them to url's before merging inot master branch.

### CODE REVIEW

File: cluster/azure/aks-gitops/variables.tf [hunk 1]
```diff
@@ -102,11 +102,6 @@ variable "kubeconfig_filename" {
   default     = "bedrock_kube_config"
 }
 
-variable "kubeconfig_recreate" {
```
Comment (jmspring) [inline cluster/azure/aks-gitops/variables.tf]: With the removal of this, if you do an update to the cluster does the config get recreated?

Comment (sarath-p) [reply cluster/azure/aks-gitops/variables.tf]: No, the user would need to delete the file and run `terraform apply` again to recreate the file with updated credentials which is similar to the current behavior. Currently, user changes the value `kubeconfig_recreate`and run `terraform apply` to recreate the file.

Comment (jmspring) [reply cluster/azure/aks-gitops/variables.tf]: Then this will require documenting.

Comment (sarath-p) [reply cluster/azure/aks-gitops/variables.tf]: Good idea. Added a note on how to recreate/re-download the file.

**Pull request merged**